tests/unit/processor/field_manager/test_source_target_rule.py

```diff
@@ -0,0 +1,109 @@
+# pylint: disable=protected-access
+# pylint: disable=missing-docstring
+import pytest
+from logprep.processor.base.exceptions import InvalidRuleDefinitionError
+from logprep.processor.field_manager.rule import FieldManagerRule
+
+
+class TestSourceTargetRule:
```

Comment:
Can you rename this test class and the file. It should contain `filed manager` and not `source target`.

Reply:
done